src/c/types/sequence.c

```diff
@@ -164,6 +164,24 @@ bool deserialize_sequence_double(MicroBuffer* buffer, double* array, uint32_t* s
     return deserialize_array_double(buffer, array, *size);
 }
 
+bool serialize_endian_sequence_char(MicroBuffer* buffer, const Endianness endianness, const char* array, const uint32_t size)
+{
+    serialize_endian_uint32_t(buffer, endianness, size);
+    return serialize_array_char(buffer, array, size);
+}
+
+bool serialize_endian_sequence_bool(MicroBuffer* buffer, const Endianness endianness, const bool* array, const uint32_t size)
+{
+    serialize_endian_uint32_t(buffer, endianness, size);
+    return serialize_array_bool(buffer, array, size);
+}
+
+bool serialize_endian_sequence_uint8_t(MicroBuffer* buffer, const Endianness endianness, const uint8_t* array, const uint32_t size)
+{
+    serialize_endian_uint32_t(buffer, endianness, size);
+    return serialize_array_uint8_t(buffer, array, size);
+}
+
 bool serialize_endian_sequence_uint16_t(MicroBuffer* buffer, const Endianness endianness, const uint16_t* array, const uint32_t size)
 {
     serialize_endian_uint32_t(buffer, endianness, size);
```

Comment:
This function is not safe. First call can fail but second not. Example: Array of one element when buffer is only 3 bytes left.

Reply:
Done!